node/node.go

```diff
@@ -400,5 +405,22 @@ func (n *Node) healthStatusHandler(err error) {
 		if err = n.pubsubServer.PublishWithEvents(n.ctx, healthStatusEvent, map[string][]string{events.EventNodeTypeKey: {events.EventHealthStatus}}); err != nil {
 			panic(err)
 		}
+
 	}
 }
+
+func (n *Node) startPrometheusServer() {
+	go func() {
+		http.Handle("/metrics", promhttp.Handler())
+		srv := &http.Server{
+			Addr:         ":2112",
+			ReadTimeout:  5 * time.Second,
+			WriteTimeout: 10 * time.Second,
+			Handler:      http.DefaultServeMux,
+		}
+		if err := srv.ListenAndServe(); err != nil {
+			// Error starting or closing listener:
+			n.Logger.Error("Prometheus HTTP server ListenAndServe", "err", err)
```

Comment:
here seems like we swallow the error - i.e the sequencer will start even if the metric server will not which IMO may be confusing for the user.
Why not catch the error, return to the caller, and have the caller panic on it?